packages/swapper/src/swappers/zrx/ZrxSwapper.ts

```diff
@@ -84,14 +84,16 @@ export class ZrxSwapper<T extends EvmSupportedChainIds> implements Swapper<T> {
     const { assetIds = [], sellAssetId } = args
     return assetIds.filter(
       (id) =>
-        id.startsWith(this.chainId) &&
-        sellAssetId?.startsWith(this.chainId) &&
+        id.startsWith(`${this.chainId}/`) &&
```

Comment:
use `fromAssetId` and use `chainId` rather than string interpolation here

Reply:
https://github.com/shapeshift/lib/pull/1157/commits/461e39dc1beeb2f5088d2e26a54f6c2d200b3a21